### PR TITLE
Add board test

### PR DESCRIPTION
**Pull request by Rostammahabadi**
Added board tests and validating coordinates

### CODE REVIEW

File: test/board_test.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/cell'
+require './lib/ship'
+require '.lib/board'
```
Comment (Gallup93) [inline test/board_test.rb]: missing "/" in last require statement

File: test/board_test.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+    assert_equal Hash, board.class
+  end
+
+  def test_validating_coordinates
+    board = Board.new
+
+    assert_equal true, board.valid_coordinate?("A1")
+    assert_equal true, board.valid_coordinate?("D4")
+  end
```
Comment (Gallup93) [inline test/board_test.rb:27]: missing "end" for class

File: test/board_test.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    board.cells
+
+    assert_equal Hash, board.class
+  end
+
+  def test_validating_coordinates
+    board = Board.new
+
+    assert_equal true, board.valid_coordinate?("A1")
+    assert_equal true, board.valid_coordinate?("D4")
```
Comment (Gallup93) [inline test/board_test.rb:26]: I was thinking we use :symbols for the coordinate name since they are something we wont be changing over time. Thoughts?

Comment (Rostammahabadi) [reply test/board_test.rb:26]: If you check out iteration 2, i'm matching what they gave us as the test/return.

**Pull request merged**